fearless/Modules/Staking/StakingMain/StakingMainPresenter.swift

```diff
@@ -296,6 +296,10 @@ extension StakingMainPresenter: StakingMainPresenterProtocol {
         }
         wireframe.showRedeem(from: view)
     }
+
+    func networkInfoViewDidCollapsed(_ value: Bool) {
+        networkInfoViewModelFactory.viewDidCollapsed(value)
```

Comment:
Function name is a little bit confusing to me. I suggest to rename it to something like networkInfoViewDidChangeExpansion(isExpanded: Bool)